docs/developer-guide/running-locally.md

```diff
@@ -22,7 +22,7 @@ kubectl apply -n argocd --force -f manifests/install.yaml
 Make sure that ArgoCD is not running in your development cluster by scaling down the deployments:
 
 ```shell
-kubectl -n argocd scale statefulset/argocd-application-controller --replicas 0
+kubectl -n argocd scale deployment/argocd-application-controller --replicas 0
```

Comment:
The controller has been changed to a statefulset as of v1.8, so this instruction is incorrect for v1.8.